src/SprykerEco/Zed/Heidelpay/Business/Basket/BasketHandler.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Heidelpay\Business\Basket;
+
+use Generated\Shared\Transfer\HeidelpayBasketRequestTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use SprykerEco\Zed\Heidelpay\Business\Adapter\Basket\BasketInterface;
+
+class BasketHandler implements BasketHandlerInterface
```

Comment:
I thinks model is called `BasketCreator`

Reply:
Don't forget then change interface name too

Reply:
Done